frontend/src/components/GoalForm/ResourceRepeater.js

```diff
@@ -38,7 +38,7 @@ export default function ResourceRepeater({
         <p className="usa-prose text-bold margin-bottom-0">Resource links</p>
         <ul className="usa-list usa-list--unstyled">
           {onlyResourcesWithValues.map((resource) => (
-            !(status === 'Complete' && goalStatus === 'Closed') || resource.onAnyReport ? (
+            !(status === 'Complete' || goalStatus === 'Closed') || resource.onAnyReport ? (
```

Comment:
Not sure if this should be mentioned in testing.

Reply:
do you mean in the PR notes?

Reply:
Yea just for testing purposes.